examples/install/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  vault.hcl: "\ndisable_mlock = true\nui = true\nlog_level = \"Debug\"\nlistener \"tcp\"
```

Comment:
No, I copied the end ConfigMap generated by `relay`. Let me see if I can clean up the presentation.
(I'll probably want to default this if possible, rather than require a default configuration, but this is probably fine initially)

Reply:
Yeah, that worked like a charm.